targets/TARGET_Cypress/scripts/PSOC6.py

```diff
@@ -402,6 +404,9 @@ def parse_args():
     sign_subcommand.add_argument(
         "--artefacts-location", required=True, help="the path to the application artefacts."
     )
+    sign_subcommand.add_argument(
+        "--application-name", required=True, help="Name of the application."
```

Comment:
```suggestion
        "--application-name", required=True, help="name of the application."
```
To be uniform with the other argument.

Reply:
Done, not relevant anymore after bringing back `--m4hex` and `--elf`